node_modules/oae-preview-processor/lib/processors/link/vimeo.js

```diff
@@ -62,7 +62,7 @@ var generatePreviews = module.exports.generatePreviews = function(ctx, contentOb
 
         // Ignoring this video if it has no thumbnail.
         if (_.isEmpty(info) || !info[0].thumbnail_medium) {
-            return callback(null, false);
+            return callback(null, true);
```

Comment:
Because it was wrong; if the video has no thumbnail, we want to ignore it, and to ignore it we need to set `ignore` to `true`.